### PR TITLE
snap: use "SSO Password:" in the `snap login` prompt

### PR DESCRIPTION
**Pull request by mvo5**
This avoids potential confusion with the sudo password prompt when
`sudo snap login` is run.

LP: #1637299

### CODE REVIEW

**Comment by niemeyer** [comment]
-1

"SSO" is no better for

**Comment by niemeyer** [comment]
Jesus..

File: cmd/snap/cmd_login.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func requestLoginWith2faRetry(email, password string) error {
 }
 
 func requestLogin(email string) error {
-	fmt.Fprint(Stdout, i18n.G("Password: "))
+	fmt.Fprint(Stdout, i18n.G("SSO Password: "))
```
Comment (niemeyer) [inline cmd/snap/cmd_login.go]: How about `"Store password: "`?

Comment (mvo5) [inline cmd/snap/cmd_login.go]: I like that! Much nicer than SSO password.

**Comment by chipaca** [comment]
I came here to shout that SSO means nothing to anybody, but see that Gustavo already caught that one. Phew.

However, how would somebody new know that "Store password" is not wanting a yes/no answer?

Is there a reason "Ubuntu One password" is bad?

**Comment by niemeyer** [comment]
Good point about store being a verb as well. Not sure if this will create problems or not, but maybe we can think of something better. How about `("Password of %s: ", email)`?

I was avoiding "Ubuntu One" in anticipation of custom stores.

**Pull request merged**